tests/integration/users/test_viewset.py

```diff
@@ -85,7 +85,6 @@ def test_simple_creation(self):
 
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         new_user = User.objects.get(username='Test name keimeiShieX4Aekoe3do')
-        self.assertEqual(response.data['id'], new_user.id)
```

Comment:
There is no assert statement that uses new_user. Could you either add one or add a comment, that no assert statement is needed, because it only testes, that the call does not rais a DoNotExists exception.

Reply:
If you don't want to use the varialbe new_user, then please remove it.